lua/cfc_notifications/shared/sh_base.lua

```diff
@@ -34,7 +34,7 @@ if SERVER then
     end )
 
     -- Give chance for other addons to load
-    timer.Simple( 0, function()
+    hook.Add( "Initialize", "cfc_notifications_init", function()
```

Comment:
your hook doesnt follow the cfc style guidelines 😲

Reply:
ah heck, this project was made before those existed lol

Reply:
it'll need a few changes, i can make an issue for it if you're ok with merging as it is rn?

Reply:
🚓